react/Trigger.tsx

```diff
@@ -47,11 +57,35 @@ function Trigger(props: Props) {
   }
 
   const handleClick = (e: React.MouseEvent<TriggerElement>) => {
+    if (trigger !== 'click') {
+      return
+    }
+
     e.stopPropagation()
```

Comment:
Why is that needed?

Reply:
Because I'm committing something to be done with this event already, so it seems to me that if I let the event go further some parent element might get the event and do something too (like opening a modal, going to some link, etc)